modules/kmm-example-module-cr.adoc

```diff
@@ -81,7 +81,10 @@ spec:
 <7> For any other kernel, build the image using the Dockerfile in the `my-kmod` ConfigMap.
 <8> Optional.
 <9> Optional: A value for `some-kubernetes-secret` can be obtained from the build environment at `/run/secrets/some-kubernetes-secret`.
-<10> Optional: Avoid using this parameter. If set to `true`, the build is allowed to pull the image in the Dockerfile `FROM` instruction using plain HTTP.
+<10> Optional: Avoid using this parameter.  If set to `true`, the build is allowed to pull the image in the
+Dockerfile `FROM` instruction using plain HTTP.
++
+When building kmod images or signing kmods within a kmod image, you might sometimes need to pull base images from a registry that serves a certificate signed by an untrusted Certificate Authority (CA). In order for KMM to trust that CA, it must also trust the new CA by replacing the cluster's CA bundle.
```

Comment:
@qbarrand I added this text to the existing text. Should I remove the existing text?

Reply:
Yes please.

Reply:
Removed existing text.